stockflux-securities-master/src/components/ToolTip.js

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import "./ToolTip.css";
+import PropTypes from "prop-types";
+
+const ToolTip = ({ children, message, className }) => {
+  return (
+    <div className={`tool-tip-container ${className}`}>
```

Comment:
This adds `undefined` class to the class list as shown below:
![image](https://user-images.githubusercontent.com/32611900/62789038-03b21380-bac0-11e9-993f-7299775497f7.png)

Reply:
I've changed this to `<div className={`tool-tip-container ${className ? className : ""}`}>` which fixes this. I've also done this to the `Alert`, `Button` and `Confirmation` components which also have has optional `className` props.